### PR TITLE
Calem branch

### PR DESCRIPTION
**Pull request by JoaoCalem**
Should I be putting my own data in the data folder?

### CODE REVIEW

File: data/day_7/input.txt [hunk 1]
```diff
@@ -0,0 +1 @@
...6,533,28,391,39,857,842,601,1216,151,538,900,481,39,363,400,78,1006,384,957,626,1503,269,429,259,755,1140,537,45,1,243,194,815,346,376,304,1167,423,822,166,282,20,200,133,892,140,1073,639,409,1644,105,828,881,1155,41,1590,647,449,89,845,554,411,260,837,237,377,154,49,36,687,62,230,14,11,26,853,497,48,130,17,1817,131,433,106,414,1034,146,977,565,670,581,999,4,164,706,58,212,554,101,64,123,219,154,28,531,626,64,782,505,364,281,1084,1414,27,226,736,507,837,142,434,319,409,819,48,23,820,454,1080,567,783,201,53,1153,404,187,814,28,898,309,380,295,437,366,1226,861,100,550,119,8,233,1452,30,753,381,365,683,474,227,155,1795,1093,208,159,121,1547,942,194,139,325,838,901,597,805,486,1499,597,405,1042,41,222,740,478,884,165,62,547,15,20,309,454,132,5,136,972,45,65,267,463,581,244,51,981,1120,75,410,1310,991,607,1801,950,913,98,769,81,322,19,459,1210,617,48,1163,1243,332,92,1713,120,269,20,246,323,1365,80,1165,1344,236,750,269,62,375,143,163,1444,172,789,521,105,369,1,40,341,625,674,1073,1376,108,391,277,122,181,134,78,1058,103,300,30,848,708,59,93,204,263,32,951,1884,23,488,258,1156,838,72,310,29,209,908,39,119,119,352,70,139,3,884,86,57,541,1553,5,105,710,29,335,1197,1152,144,696,685,948,563
```
Comment (rodloboz) [inline data/day_7/input.txt]: Can we remove data inputs? Solution should work with any AOC data, hence should be valid with existing Day7 data

**Review by rodloboz — CHANGES_REQUESTED** [review]
Can be merged once new data input if removed (I have some Ruby tests against the existing dataset)

File: python/day_7/tests/test_calem.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import unittest
+from crabs import fuel_cost, enhanced_fuel_cost
```
Comment (rodloboz) [inline python/day_7/tests/test_calem.py]: Should you import from calem? This is importing the existing solution I think

Comment (JoaoCalem) [reply python/day_7/tests/test_calem.py]: Yeah, nice catch ! Fixed it

File: data/day_7/input.txt (hunk 1 above)
Comment (JoaoCalem) [reply data/day_7/input.txt]: Removed it now, but I don't think day 7 data is on the master branch. That's why I added my own originally

**Review by rodloboz — APPROVED** [review]

**Comment by rodloboz** [comment]
Good to merge 👍

**Pull request merged**